vendor/vendor.json

```diff
@@ -15,44 +15,90 @@
 			"versionExact": "v0.11.5"
 		},
 		{
-			"checksumSHA1": "MfMZSsn79Ey3/+Zn2UX1mgVGU/A=",
+			"checksumSHA1": "ZfId5ZYn7jmL0xTxooHcxrLlXoA=",
 			"path": "github.com/newrelic/infra-integrations-sdk/args",
-			"revision": "baf70b67539dc2821d60b5810639b6bc188b86f8",
-			"revisionTime": "2017-10-11T10:26:32Z",
-			"version": "v2.0.0",
-			"versionExact": "v2.0.0"
+			"revision": "ca4f7271a7841fa2d613276bad5907e5e1eddc78",
+			"revisionTime": "2018-11-29T09:11:08Z",
+			"version": "=v3.1.0",
+			"versionExact": "v3.1.0"
 		},
 		{
 			"checksumSHA1": "45zcAovsGwfEvjiql/9SI1JWArE=",
 			"path": "github.com/newrelic/infra-integrations-sdk/cache",
 			"revision": "baf70b67539dc2821d60b5810639b6bc188b86f8",
 			"revisionTime": "2017-10-11T10:26:32Z",
-			"version": "v2.0.0",
-			"versionExact": "v2.0.0"
+			"version": "=v3.1.0",
+			"versionExact": "v3.1.0"
+		},
+		{
+			"checksumSHA1": "KaZLSd/JUZz/mqfW3jFd1j2QdWA=",
+			"path": "github.com/newrelic/infra-integrations-sdk/data/event",
+			"revision": "ca4f7271a7841fa2d613276bad5907e5e1eddc78",
+			"revisionTime": "2018-11-29T09:11:08Z",
+			"version": "=v3.1.0",
+			"versionExact": "v3.1.0"
+		},
+		{
+			"checksumSHA1": "EOoxXGqFDPkCvj7wec2S5kCXY2I=",
+			"path": "github.com/newrelic/infra-integrations-sdk/data/inventory",
+			"revision": "ca4f7271a7841fa2d613276bad5907e5e1eddc78",
+			"revisionTime": "2018-11-29T09:11:08Z",
+			"version": "=v3.1.0",
+			"versionExact": "v3.1.0"
+		},
+		{
+			"checksumSHA1": "39SDqHpLpE9J1nrIwYI+8ukzXJs=",
+			"path": "github.com/newrelic/infra-integrations-sdk/data/metric",
+			"revision": "ca4f7271a7841fa2d613276bad5907e5e1eddc78",
+			"revisionTime": "2018-11-29T09:11:08Z",
+			"version": "=v3.1.0",
+			"versionExact": "v3.1.0"
 		},
 		{
-			"checksumSHA1": "gRz6v46pNwphQXmd9rSP05I7cHo=",
+			"checksumSHA1": "Q23NjXVh6j31N6ZChLG/FNv7fwU=",
+			"path": "github.com/newrelic/infra-integrations-sdk/integration",
+			"revision": "ca4f7271a7841fa2d613276bad5907e5e1eddc78",
+			"revisionTime": "2018-11-29T09:11:08Z",
+			"version": "=v3.1.0",
+			"versionExact": "v3.1.0"
+		},
+		{
+			"checksumSHA1": "6/DhD+/LhC/k3NaLgkpd6iE8b5E=",
 			"path": "github.com/newrelic/infra-integrations-sdk/log",
-			"revision": "baf70b67539dc2821d60b5810639b6bc188b86f8",
-			"revisionTime": "2017-10-11T10:26:32Z",
-			"version": "v2.0.0",
-			"versionExact": "v2.0.0"
+			"revision": "ca4f7271a7841fa2d613276bad5907e5e1eddc78",
+			"revisionTime": "2018-11-29T09:11:08Z",
+			"version": "=v3.1.0",
+			"versionExact": "v3.1.0"
 		},
 		{
 			"checksumSHA1": "/OBil2T0jjmaRvLhTCsMXLVG2N4=",
 			"path": "github.com/newrelic/infra-integrations-sdk/metric",
```

Comment:
check this metric pkg. It is not a package for the v3.1.0 anymore. It's in data directory